examples/simple-service-poc/simple_service.py

```diff
@@ -146,21 +143,15 @@ async def main(
             ],
             expiration=datetime.now(timezone.utc) + timedelta(minutes=120),
         )
-
-        # helper functions to display / filter instances
-
-        def instances():
-            return [f"{s.name}: {s.state.value} on {s.provider_name}" for s in cluster.instances]
+        instances = cluster.instances
 
         def still_starting():
-            return len(cluster.instances) < num_instances or any(
-                s.state == ServiceState.starting for s in cluster.instances
-            )
+            return any(i.state in (ServiceState.pending, ServiceState.running) for i in instances)
```

Comment:
hmm... have you tested it? ... we meant to pull the _starting_ ones so how would _running_ make sense here?

Reply:
937b39c